### PR TITLE
OffroadHome: separate the update and alert buttons. 

### LINKED ISSUE #21266: Update alert incorrectly supersedes overtemp alert
**Describe the bug**

When attempting to go onroad with a hot device that has an update available, the update alert displays instead of the overtemp alert. It keeps displaying until the device cools down, at which point it goes onroad.

**How to reproduce or log data**

1) Have an update available
2) Be hot
3) Start car
4) Observe device correctly refuse to go onroad until it cools down, but fail to properly tell the user why

**Expected behavior**

In this situation, the overtemp alert should supersede the update available alert.

**Device/Version information (please complete the following information):**
 - Dongle ID: 7aae592fc08e895f
 - Route: 7aae592fc08e895f|2021-06-14--16-20-37
 - Timestamp: beginning of the route, and prior
 - Version: day-old master + specialsauce (I was still running the branch from #21253)
 - Car make/model: 2018 Volkswagen Golf R

### PR DESCRIPTION
**Pull request by deanlee**
fix issue: #21266

### CODE REVIEW

File: selfdrive/ui/qt/widgets/offroad_alerts.cc [hunk 1]
```diff
@@ -61,52 +61,61 @@ OffroadAlert::OffroadAlert(QWidget* parent) : QFrame(parent) {
       border-radius: 30px;
       background-color: white;
     }
+    #rebootBtn {
+      background-color: #E22C2C;
+    }
```
Comment (adeebshihadeh) [inline selfdrive/ui/qt/widgets/offroad_alerts.cc:66]:
```suggestion
```

**Comment by deanlee** [comment]
think it's better to split the update from OffroadAlert into a separate widget

**Comment by adeebshihadeh** [comment]
I'm fine with that

**Comment by adeebshihadeh** [comment]
Noticed some regressions:
* missing stretch at the bottom of the alerts layout
* update window doesn't go away when UpdateAvailable is deleted (button goes away though)

Used `selfdrive/ui/tests/cycle_offroad_alerts.py` to test.

**Comment by deanlee** [comment]
fixed

**Comment by adeebshihadeh** [comment]
There's a flash on startup now while the layout resizes. I think it's only if alerts exist on startup.

**Pull request merged**